app/views/layouts/_logo.html.erb

```diff
@@ -1,21 +1,18 @@
 <a href="<%= root_path %>" class="site-logo" aria-label="<%= t("views.main.aria_home", community: community_name) %>">
-  <!-- its a little explicit for the moment but it will make
-      it easier to remove code along with the FeatureFlag -->
-  <% if FeatureFlag.enabled?(:creator_onboarding) %>
-    <% if Settings::General.resized_logo.present? %>
-      <img class="site-logo__img" src="<%= Settings::General.resized_logo %>" alt="<%= community_name %>">
-    <% else %>
-      <span class="site-logo__community-name truncate-at-2">
-        <%= community_name %>
-      </span>
-    <% end %>
+  <!-- This is a little explicit at the moment, but it
+      will make it easier to remove any SVG-related code -->
+  <% if Settings::General.resized_logo.present? %>
+    <img class="site-logo__img" src="<%= Settings::General.resized_logo %>" alt="<%= community_name %>">
   <% else %>
-    <% if Settings::General.logo_svg.present? %>
-      <%= logo_svg %>
-    <% else %>
-      <span class="site-logo__community-name truncate-at-2">
-        <%= community_name %>
-      </span>
-    <% end %>
+    <span class="site-logo__community-name truncate-at-2">
+      <%= community_name %>
+    </span>
+  <% end %>
+  <% if Settings::General.logo_svg.present? %>
```

Comment:
@juliannatetreault lets remove this entire block as well, and then we'll go in and remove all other traces of "logo_svg" in the other issue that you've logged.

Reply:
Done!